### PR TITLE
[homematic] add description for window/door contact

### PR DESCRIPTION
**Pull request by TheTrueRandom**
I added more HomeMatic IP devices to the descriptions

### CODE REVIEW

**Comment by MHerbst** [comment]
LGTM

File: bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties [hunk 1]
```diff
@@ -3,14 +3,14 @@ HM-RC-4-2=Funk-Handsender 4 Tasten
 HM-LC-Sw1-Pl-2=Funk-Schaltaktor 1-fach, Zwischenstecker
 HM-Sec-SD-2=Funk-Rauchmelder
 HM-WDS30-OT2-SM-2=Funk-Temperaturdifferenzsensor
-HM-Sen-MDIR-O-2=Funk-Bewegungsmelder au�en
```
Comment (J-N-K) [inline bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties:6]: Has the encoding for this file changed?

**Comment by TheTrueRandom** [comment]
@J-N-K I changed the file encoding back to ISO-8859-1. I still wonder though why in the year 2020 we are still not using UTF-8 for our code.. It took me 30 minutes to configure IntelliJ to use ISO-8859-1 for this file and not mess up my other projects

**Comment by chrostek** [comment]
oh, saw your PR too late - your device is also included in my PR #7911

**Comment by cpmeister** [comment]
Thanks for your contribution @TheTrueRandom . I don't want to discourage you from making future PRs but since PR #7911 already included the changes from this PR and @chrostek would have to perform extra work to resolve conflicts that merging your PR would cause, I'm going to close this PR and merge #7911 instead.

@TheTrueRandom, just as a reminder that these were strange circumstances where two PRs for the same changes came in at the same time so please don't think I'm trying to discourage you from making future contributions to this project.

Thanks

**Comment by TheTrueRandom** [comment]
sure, not a problem :smile: happy to see some homematic action

**Pull request closed without merging**